README.md

```diff
@@ -36,16 +36,21 @@ For usage information run `beet [subcommand] -h`.
 
 ## Configuration
 
+
+The other plugins (utaitedb and touhoudb) will use the same settings as vocadb as a fallback, so you don't have to repeat yourself.
+
 ```yaml
 vocadb: # Name of the plugin you want to configure (vocadb, utaitedb or touhoudb)
   source_weight: 0.5 # Penalty to be added to all matches when using autotagger (0 disabled, 1 highest)
   prefer_romaji: false # Prefer romanized if they exist rather than Japanese
   translated_lyrics: false # Always get translated lyrics if they're available
   include_featured_album_artists: false # Include featured artists in album artists string
   va_string: "Various artists" # Album artist name to use when artist list contains many artists
+  max_results: 5 # Increase it if a song or album your're looking for doesn't show up in the list of candidates
```

Comment:
```suggestion
  max_results: 5 # Number of results to get from source. Consider increasing if correct song or album doesn't show up in the list of candidates
```

Mainly just wording preference, this is how I would write it.
There's a typo in "your're" that should be fixed if you don't want to merge this suggestion.